### ISSUE TITLE
Arguments not passed to field

### ISSUE THREAD

**Issue by fubar-coder** [graphql-dotnet/graphql-dotnet#109]
```csharp
Field<CompanyType>()
  .Name("company")
  .Argument<IntGraphType>("id", "Record ID")
  .Argument<IntGraphType>("skip", "Records to skip")
  .Argument<IntGraphType>("take", "Records to take")
  .Resolve(context =>
  {
    var skip = context.GetArgument<int?>("skip");
    var take = context.GetArgument<int?>("take");
  });
```

Calling the query using variables results in `skip` and `take` to always be `null`.

This is the query I used:
```
query q($skip: Int, $take: Int) {
  company(skip: $skip, take: $take) {
    id
  }
}
```

The arguments are definitely passed to the document excutor.

Everything works as expected when I change the field definition to the "old" field definition method.

EDIT: Tested with 0.9.1

**Comment by joemcbride** [comment]
Related to https://github.com/graphql-dotnet/graphql-dotnet/issues/74

**Comment by joemcbride** [comment]
This should be fixed in [0.9.2](https://github.com/graphql-dotnet/graphql-dotnet/releases/tag/v0.9.2).
